src/spaceObjects/beamEffect.cpp

```diff
@@ -187,3 +189,9 @@ void BeamEffect::setTarget(P<SpaceObject> target, sf::Vector2f hitLocation)
     sf::Vector3f targetPos(target->getPosition().x, target->getPosition().y, 0);
     hitNormal = sf::normalize(targetPos - hitPos);
 }
+
+glm::mat4 BeamEffect::getModelMatrix() const
+{
+    auto position = const_cast<BeamEffect*>(this)->getPosition();
```

Comment:
guess the const cast is no longer needed now? :)

Reply:
ha, indeed!